firefox-ios/Storage/Rust/RustRemoteTabs.swift

```diff
@@ -175,140 +177,162 @@ public class RustRemoteTabs {
 
     // MARK: Remote Command APIs
     public func addRemoteCommand(deviceId: String, url: URL) {
-        DispatchQueue.global().async { [unowned self] in
-            guard let commandQueue = self.commandQueue else {
+        commandQueue.async { [unowned self] in
+            guard let tabsCommandQueue = self.tabsCommandQueue else {
```

Comment:
One other advantage to using `self.queue`: this would be an unsynchronized access to `self.tabsCommandQueue`.

`self.tabsCommandQueue` is initialized [on `queue`](https://github.com/mozilla-mobile/firefox-ios/blob/1606a5e49105d75bd12281e8eaf8cf6cc5f9a8ee/firefox-ios/Storage/Rust/RustRemoteTabs.swift#L54-L58)—the `queue.sync` there is emulating a mutex—but this is accessing it on `commandQueue`.

(I think I typed the word "queue" too many times in that last sentence! 😅)